_data/compose-cli/docker_compose_pull.yaml

```diff
@@ -49,10 +53,52 @@ options:
   default_value: "false"
   description: Pull without printing progress information
   deprecated: false
+  hidden: false
   experimental: false
   experimentalcli: false
   kubernetes: false
   swarm: false
+examples: |-
+  suppose you have this `compose.yaml`:
```

Comment:
We should look at;

- Adding a heading for each example
- Perhaps split the example into `docker compose pull` (without service specified) and one with a service (and adjust the compose file to have multiple services that use an image).